stable/fluentd/templates/configmap.yaml

```diff
@@ -27,3 +27,11 @@ data:
       @type prometheus_output_monitor
     </source>
 {{- end }}
+{{- if .Values.plugins.enabled }}
+  install-plugins.sh: |-
+    #!/bin/sh
+  {{- range $plugin := .Values.plugins.pluginsList }}
+    fluent-gem install {{ $plugin }}
+  {{- end }}
+    exec /run.sh
```

Comment:
What if the list of plugins is empty? We should check that scenario

Reply:
This does manage that, if the list is empty but this is enabled. It'll just run the `run.sh` script.

Reply:
I will check if the list is empty and the plug-in is enabled to avoid running an empty bash script

Reply:
I think this is excessive as we have the `enabled` property but for sake of argument, I've added a check for an empty list. Please